src/cuga/backend/knowledge/mcp_server.py

```
@@ -88,6 +95,7 @@ def _get_client() -> httpx.AsyncClient:
         _client = httpx.AsyncClient(
             base_url=CUGA_BACKEND_URL,
             timeout=httpx.Timeout(60.0, connect=10.0),
+            verify=False,
```

Ruff (0.15.12)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)